exchange/exchange.go

```diff
@@ -119,9 +124,25 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 		e.me.RecordImps(impLabels)
 	}
 
+	// Make our best guess if GDPR applies
+	usersyncIfAmbiguous := e.UsersyncIfAmbiguous
+	var geo *openrtb.Geo = nil
+	if bidRequest.User != nil && bidRequest.User.Geo != nil {
+		geo = bidRequest.User.Geo
+	} else if bidRequest.Device != nil && bidRequest.Device.Geo != nil {
+		geo = bidRequest.Device.Geo
+	}
+	if geo != nil {
+		if _, found := e.eeaCountries[geo.Country]; found {
```

Comment:
Is capitalization a concern? Should we convert the geo country to all caps before checking? .. or add both caps and lower case countries to the list?

Reply:
That is a good question.

Reply:
I escalated this question to legal. I think we're fine to leave it as upper case for now as that's whats listed in the ISO-3166-1-alpha-3 spec.